yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/retrieve/GetSessionResult.java

```diff
@@ -78,6 +78,10 @@ public List<FaceMatchCheckResponse> getFaceMatchChecks() {
         return filterChecksByType(FaceMatchCheckResponse.class);
     }
 
+    public List<FaceComparisonCheckResponse> getFaceComparisonChecks() {
```

Comment:
Can we add/update a unit test for this please?

Reply:
Added